build-tools/automation/azure-pipelines.yaml

```diff
@@ -1361,8 +1361,13 @@ stages:
       yamlResourceName: yaml
       dependsOn: signing
       artifactName: nuget-signed
+      # NOTE: don't download API 32 preview packs or Darwin packs, so we don't insert them into VS
       artifactPatterns: |
-        !*Darwin*
+        Microsoft.Android.31.*.nupkg
+        Microsoft.Android.Sdk.BundleTool.*.nupkg
+        Microsoft.Android.Sdk.Windows.*.nupkg
+        Microsoft.Android.Templates.*.nupkg
+        Microsoft.NET.Sdk.Android.*.nupkg
```

Comment:
NOTE: two `!*Darwin*` and `!*32*` won't currently work -- there is a bug in AzDO.

Reply:
For reference: https://developercommunity.visualstudio.com/t/downloadpipelineartifact-4/1434766#T-N1442966